pages/deployment/multicontainer.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Multicontainer Guide
+excerpt: A guide to building multicontainer applications with resin.io
+---
+
+# Multicontainer Guide
+
+As your applications grow more complex, you may find significant benefit in running some services in separate containers. Multicontainer applications allow you to better isolate, duplicate, and manage key services, making your applications more modular and secure. Starting with resinOS vx.x.x, resin.io provides multicontainer application support, built around the [Docker Compose][docker-compose] tool. This guide will cover the considerations you need to take into account when using a multicontainer instance, including Docker Compose configuration, important resin.io specific settings, and single container applications.
+
+## Docker Compose
```

Comment:
we should link to a full reference of all the things we currently support. I know @pcarranzav has a compiled list in a spread sheet